### ISSUE TITLE
After deleting the last calender item there is only a white page

### ISSUE THREAD

**Issue by Feuer-sturm** [ioBroker/ioBroker.fullcalendar#55]
**Describe the bug**  
After deleting the last calender item there is only a white page

**To Reproduce**  
1. Go to menu Kalender
2. Add one single item (it is only one item in the calender)
3. Delete this one single entry
4. Calender is not visible any more and there is only a white page

**Expected behavior**  
Calender is still visible

**Screenshots & Logfiles**  
![Animation4](https://user-images.githubusercontent.com/25422320/227025804-8e088f69-c5c2-494e-afe6-fb6fa4a250f8.gif)

```
iobroker.fullcalendar@2.0.6 using theme "colored" index.jsx:16:8

██╗ ██████╗ ██████╗ ██████╗  ██████╗ ██╗  ██╗███████╗██████╗ 
██║██╔═══██╗██╔══██╗██╔══██╗██╔═══██╗██║ ██╔╝██╔════╝██╔══██╗
██║██║   ██║██████╔╝██████╔╝██║   ██║█████╔╝ █████╗  ██████╔╝
██║██║   ██║██╔══██╗██╔══██╗██║   ██║██╔═██╗ ██╔══╝  ██╔══██╗
██║╚██████╔╝██████╔╝██║  ██║╚██████╔╝██║  ██╗███████╗██║  ██║
╚═╝ ╚═════╝ ╚═════╝ ╚═╝  ╚═╝ ╚═════╝ ╚═╝  ╚═╝╚══════╝╚═╝  ╚═╝
Prompt.js:15:12
Nice to see you here! :) Join our dev community here https://github.com/ioBroker/ioBroker or here https://github.com/iobroker-community-adapters Prompt.js:16:12
Help us to create open source project with reactJS! Prompt.js:17:12
See you :) Prompt.js:18:12
[2023-03-22T20:12:54.196Z] Try to connect socket.io.js:66:41
TypeError: e.event is undefined
    Td EventDialog.js:77
    React 8
    V scheduler.production.min.js:13
    T scheduler.production.min.js:14
    36813 scheduler.production.min.js:14
    Webpack 12
react-dom.production.min.js:189:29
Uncaught TypeError: e.event is undefined
    Td EventDialog.js:77
    React 8
    V scheduler.production.min.js:13
    T scheduler.production.min.js:14
    36813 scheduler.production.min.js:14
    Webpack 12

```

**Versions:**  
 - Adapter version: Admin 6.4.0, 
 - fullcalender 2.0.6

**Comment by Hirsch-DE** [comment]
Hallo,
beim Editieren eines Eintrages bekomme ich auch nur ein leeres Fenster! Dadurch ist kein Bearbeiten möglich (auch nicht löschen)!
V2.0.8
Admin 6.4.3
js-controller 4.0.24

`react-dom.production.min.js:189 TypeError: e.clone is not a function
    at Vd.formatByString (index.esm.js:124:35)
    at Vd.format (index.esm.js:121:26)
    at openTimePickerDialogue (enUS.js:38:125)
    at r.forwardRef.l.props.ampmInClock (useDesktopPicker.js:83:23)
    at DesktopTimePicker.js:49:7
    at bo (react-dom.production.min.js:167:137)
    at yi (react-dom.production.min.js:193:57)
    at bs (react-dom.production.min.js:294:275)
    at Hl (react-dom.production.min.js:280:389)
    at gl (react-dom.production.min.js:280:320)
di @ react-dom.production.min.js:189
react-dom.production.min.js:127 Uncaught TypeError: e.clone is not a function
    at Vd.formatByString (index.esm.js:124:35)
    at Vd.format (index.esm.js:121:26)
    at openTimePickerDialogue (enUS.js:38:125)
    at r.forwardRef.l.props.ampmInClock (useDesktopPicker.js:83:23)
    at DesktopTimePicker.js:49:7
    at bo (react-dom.production.min.js:167:137)
    at yi (react-dom.production.min.js:193:57)
    at bs (react-dom.production.min.js:294:275)
    at Hl (react-dom.production.min.js:280:389)
    at gl (react-dom.production.min.js:280:320)`

**Comment by GermanBluefox** [comment]
Please retest with 2.1.0

**Comment by Hirsch-DE** [comment]
Hallo,

jetzt funktioniert es.

Super!

Gruß Thomas

**Comment by Feuer-sturm** [comment]
@GermanBluefox 
My issue (first post) is not fixed with fullcalender version 2.1.1

When I reproduce my issue the following error is in the log:


```
TypeError: e is null
    event Calendar.js:358
    qd Calendar.js:358
    React 8
    y scheduler.production.min.js:13
    T scheduler.production.min.js:14
    36813 scheduler.production.min.js:14
    Webpack 12
react-dom.production.min.js:189:29
Uncaught TypeError: e is null
    event Calendar.js:358
    qd Calendar.js:358
    React 8
    y scheduler.production.min.js:13
    T scheduler.production.min.js:14
    36813 scheduler.production.min.js:14
    Webpack 12

```

**Comment by Feuer-sturm** [comment]
@GermanBluefox 
Retest with fullcalendar 2.2.0.

The issue is still active.
Log:

```
Double links: "vis.0/vis/edit.html" and "vis.0" instrument.ts:132:32
iobroker.fullcalendar@2.2.0 using theme "colored" index.jsx:16:8

██╗ ██████╗ ██████╗ ██████╗  ██████╗ ██╗  ██╗███████╗██████╗ 
██║██╔═══██╗██╔══██╗██╔══██╗██╔═══██╗██║ ██╔╝██╔════╝██╔══██╗
██║██║   ██║██████╔╝██████╔╝██║   ██║█████╔╝ █████╗  ██████╔╝
██║██║   ██║██╔══██╗██╔══██╗██║   ██║██╔═██╗ ██╔══╝  ██╔══██╗
██║╚██████╔╝██████╔╝██║  ██║╚██████╔╝██║  ██╗███████╗██║  ██║
╚═╝ ╚═════╝ ╚═════╝ ╚═╝  ╚═╝ ╚═════╝ ╚═╝  ╚═╝╚══════╝╚═╝  ╚═╝
Prompt.js:15:12
Nice to see you here! :) Join our dev community here https://github.com/ioBroker/ioBroker or here https://github.com/iobroker-community-adapters Prompt.js:16:12
Help us to create open source project with reactJS! Prompt.js:17:12
See you :) Prompt.js:18:12
[2023-05-10T19:41:54.319Z] Try to connect socket.io.js:66:41
TypeError: e is null
    event Calendar.js:390
    tv Calendar.js:390
    React 8
    y scheduler.production.min.js:13
    T scheduler.production.min.js:14
    36813 scheduler.production.min.js:14
    Webpack 12
react-dom.production.min.js:189:29
    React 9
    y scheduler.production.min.js:13
    T scheduler.production.min.js:14
    (Async: EventHandlerNonNull)
    36813 scheduler.production.min.js:14
    Webpack 12
Uncaught TypeError: e is null
    event Calendar.js:390
    tv Calendar.js:390
    React 8
    y scheduler.production.min.js:13
    T scheduler.production.min.js:14
    36813 scheduler.production.min.js:14
    Webpack 12
Calendar.js:390:42
    event Calendar.js:390
    find self-hosted:349
    tv Calendar.js:390
    React 8
    y scheduler.production.min.js:13
    T scheduler.production.min.js:14
    (Async: EventHandlerNonNull)
    36813 scheduler.production.min.js:14
    Webpack 12
```

**Comment by GermanBluefox** [comment]
Fixed in 2.2.4

**Comment by Feuer-sturm** [comment]
ok with v2.2.6
Thanks.
